### DISCUSSION TITLE
Support for Chinese fonts?

### DISCUSSION THREAD

**Discussion by xuxiaowei-com-cn** [emilk/egui#2579] [Ideas]
Smiley Sans 得意黑: From the beginning of open source on November 11, 2022 to January 13, 2023, 9.5k stars have been harvested.
GitHub：https://github.com/atelier-anchor/smiley-sans
Author: https://space.bilibili.com/38053181
Publicity video: https://www.bilibili.com/video/BV1sP411g7PZ/

**Comment by xuxiaowei-com-cn** [comment]
https://github.com/emilk/egui/pull/2580

**Reply by John-Nagle** [reply]
You can load fonts into egui.

See https://docs.rs/egui/latest/egui/struct.FontDefinitions.html

The fonts become part of the program image and may add 8 to 20 megabytes to the program size.

**Comment by xuxiaowei-com-cn** [comment]
Now I can set the font。
PR：[Add Chinese font: 得意黑 Smiley Sans](https://github.com/emilk/egui/pull/2580)
